templates/terraform/constants/cloud_run_domain_mapping.go.erb

```diff
@@ -0,0 +1,16 @@
+const domainMappingGoogleProvidedLabel = "cloud.googleapis.com/location"
+
+func domainMappingLabelDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
This field is a `TypeMap`, right? And `k` will take on the value of the size  (`labels.%`) as well as the value of each key, leading to the DSF being called n+1 times when there are n elements?

Reply:
That is correct! We need to suppress the size as well. This won't matter in any happy path since the diff for the actual key will still be present.